src/screens/TestCases/ReviewCommentMaster/Validation/AddReviewComment.js

```diff
@@ -0,0 +1,17 @@
+import * as Yup from "yup";
+import {
+  ALPHA_NUMERIC_REGEX,
+  ONLY_CHARACTER_REGEX,
+} from "../../../../settings/constants";
+export const addReviewCommentValidation = Yup.object().shape({
+  reviewer_comment: Yup.string()
+    .max(50, "Review comment title must be at most 50 characters")
+    .matches(ONLY_CHARACTER_REGEX, "Review comment title must be alphabets"),
+  remark: Yup.string()
+    .min(2, "Remark description must be at least 2 characters")
```

Comment:
what is the reason behind this line of code?

Reply:
to check the length of reviewer comment